### ISSUE TITLE
Onlyoffice cannot be reached

### ISSUE THREAD

**Issue by MrN00b00dy** [ONLYOFFICE/onlyoffice-nextcloud#119]
Hello, i am facing an issue after integrating onlyoffice to my nextcloud server, here is my setup :
- Nextcloud installed via snap in machine 1, externally accessible
- Onlyoffice doc server installed via docker in machine 2, locally accessible, https enabled with self signed cert

Here is what am up to :
- i can successfully get the onlyoffice doc server adress saved in nextcloud settings
- can create docs in nextcloud
- when opening a doc, am having the following error : « onlyoffice cannot be reached, contact admin »

PS : lately while troubleshooting, i found out that OO is working on my 5s iphone while connected to the same router as the two servers, and not working if connected via 4g. While on my iphone 6s, it is not working at all, even if connected to the router.

Am really stuck at this point.

Thanks for your help guys.

**Comment by alexanderonlyoffice** [comment]
@MrN00b00dy, we apologize for the delayed response. The problem seems to be in not trusted self-signed SSL certificates. 

If the issue is still relevant, please try to disable SSL certificates verification on Nextcloud and Document Server sides or use CA-signed certificates. 

In order to disable verification on Document Server side do the following:

1. Enter the container with the Document Server `sudo docker exec -it [container_id] bash`
2. Open /etc/onlyoffice/documentserver/default.json and find the parameter `rejectUnauthorized`
3. Change its value from **true** to **false** 
4. Restart all services of the Document Server `supervisorctl restart all`

As for Nextcloud, you need to add the following section to config.php: 
```
'onlyoffice' =>
  array (
    'verify_peer_off' => TRUE,
  )
```
Please let us know about the result. We look forward to your reply.

**Comment by shoyh1lb** [comment]
@alexanderonlyoffice 

Hello. I am in a similar situation.
Here is my setup :

- Nextcloud server and Onlyoffice server installed via VMware vSphere 

- Both are an internal nextcloud and internal onlyoffice instance, so it is not nessessary to encrypt the communication between this both applications.

![WS000008](https://user-images.githubusercontent.com/45613121/72579097-5fa88000-391b-11ea-982b-66e25d93ba59.JPG)

when opening a doc, am having the following error : « onlyoffice cannot be reached, contact admin »
How can I fix this problem?

![WS000009](https://user-images.githubusercontent.com/45613121/72579284-055bef00-391c-11ea-866e-c7fcdd0ea91c.JPG)


Thanks for your help guys.

**Comment by LinneyS** [comment]
Hi @shoyh1lb 
This error indicates that the editor is not available from the browser at address
`YOUR_NEXTCLOUD_DOMAIN/ds-vpath/web-apps/apps/api/documents/api.js`


@MrN00b00dy 
Could you please specify if the problem is resolved

**Comment by shoyh1lb** [comment]
Hi @LinneyS 
Thank you for your reply.
I don't  understand what you mean exactly.
Did you mean like below?
![picture](https://user-images.githubusercontent.com/45613121/72771395-1bcbb880-3c44-11ea-96ec-92e34883e2a7.png)

**Comment by LinneyS** [comment]
No
`/ds-vpath/` is correct setting.
Check the availability of the `YOUR_NEXTCLOUD_DOMAIN/ds-vpath/` in the browser.

**Comment by shoyh1lb** [comment]
@LinneyS 

I tried it with Firefox. It seems it's not working.
Any advice?
![WS000029](https://user-images.githubusercontent.com/45613121/72856684-9eb14980-3cfe-11ea-9461-340e023e859f.JPG)

**Comment by alexanderonlyoffice** [comment]
@shoyh1lb, please specify if you used a docker compose file to deploy ONLYOFFICE Document Server and Nextcloud. 

Specify also why there is ds-vpath (document server virtual path) in the address settings of ONLYOFFICE integration app. 

Please send me a screenshot of the 'Welcome' page of your Document Server and specify the way you installed the Document Server, send me the link to the installation guide you used.

**Comment by shoyh1lb** [comment]
@alexanderonlyoffice 
I didn't use a docker. I installed CeontOS7 server in VMware vSphere.

This is the link to the installation guide I used.
https://helpcenter.onlyoffice.com/de/server/linux/document/linux-installation-centos.aspx
I followed every steps written in the link. (Since I use port 80,I skipped "Changing the default Document Server port" tho.)

I put ds-vpath in the address referring this link.
https://help.nextcloud.com/t/connect-internal-onlyoffice-with-internal-nextcloud-with-http/56959
To be honest,I even don't know what ds-vpath is.

![WS000030](https://user-images.githubusercontent.com/45613121/72950752-9248f180-3dcf-11ea-9206-f9dd9772f921.JPG)
